homework/index.js

```diff
@@ -30,13 +30,149 @@
+          const amOrPm = upDate.getHours() < 12 ? 'AM' : 'PM';
+          const dateHours = upDate.getHours() % 12 || 12;
+          const formatedUpdate = `${upDate.getMonth()}/${upDate.getDate()}/${upDate.getFullYear()} ${dateHours}
+             : ${upDate.getMinutes()}:${upDate.getSeconds()} ${amOrPm}`;
+
+          templateElement.push(
+            '<div id="row">',
+            `${'<p id="name-info">'} Repository name : ${'<a href="'}${
+              repoInfo[repo].html_url
+            }"${'/>'} ${repoInfo[repo].name}</a></p>`,
+            `${'<p id="desc">'}  Description :  ${repoInfo[repo].description}</p>`,
+            `${'<p id="forks">'}  Forks       : ${repoInfo[repo].forks_count}</p>`,
+
+            `${'<p id="updated">'} Updated  : ${formatedUpdate}</p>`,
+            '</div>',
+          );
+
+          // Call another function(addContributors) to fill i contributor information.
+          fetchJSON(repoInfo[repo].contributors_url, (err, data) => {
+            const root = document.getElementById('root');
+
+            if (err) {
+              createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+            } else {
+              addContributors(data);
+            }
+          });
+        }
+      }
+    }
+    const htmlString = templateElement.join('');
+    document.getElementById('section1').innerHTML = htmlString;
+  }
+
+  // Create options under 'SELECT' element which should have HYF Repositories.
+
+  function loadSelectionValues(userInfo) {
+    const sortRepoName = [];
+
+    const userRepo = JSON.parse(userInfo);
+    const mySelect = document.getElementById('myselect');
+
+    // push all the HYP repo names to sort array.
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const repo in userRepo) {
+      if (Object.prototype.hasOwnProperty.call(userRepo, repo)) {
+        sortRepoName.push(userRepo[repo].name);
+      }
+    }
+
+    // sort the repo name using sort function and localeComapare for uppercase and lowercase sorting.
+    sortRepoName.sort((a, b) => a.localeCompare(b));
+
+    // Create Option under Select element and attach the  same with SELECT element.
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const repo of sortRepoName) {
+      const option = document.createElement('option');
+      option.value = repo;
+      option.text = repo;
+      mySelect.appendChild(option);
+    }
+
+    const selectBox = document.getElementById('myselect');
+    const selectedValue = selectBox.options[selectBox.selectedIndex].value;
+
+    // Load Repository information for the choose repository name in the select box.
+    loadDiv(userRepo, selectedValue);
+    selectBox.onchange = function() {
+      loadDiv(userRepo, selectBox.value);
+    };
+  }
+
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+      const myParent = document.body;
```

Comment:
Who is my? I'd call it something like `BodyEl`

Reply:
changed myParent to BodyEl.